doc/user_guide/mellon_user_guide.adoc

```diff
@@ -3012,9 +3013,9 @@ context of use.
 another attribute representation system. Such a set of
 rules is known as an “attribute profile”.
 
-SAML:: Security Assert Markup Language. 
+SAML:: Security Assertion Markup Language.
 
-Service Provider:: A service provider is a SAML relying party which
+Service Provider:: A service provider is a SAML-relying party which
```

Comment:
I think "relying party" is included here because it is frequently used interchangeably with "service provider". It may make sense to change the text to `SAML _relying party_` instead, to clarify that it is a term.

Reply:
Ah, I see. Fixed and pushed.